HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.16 / 2021-07-15
+
+- Updated dependencies
```

Comment:
You may want to also make note that you made changes to the code to accommodate the changes IdentityModel

Reply:
Sure, added